### PR TITLE
Part of #17670: Replace Typography with Text component in: detected-token-values.js

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
This PR is a part of #17670 Replace Typography with Text component for:
**ui\components\app\detected-token\detected-token-values\detected-token-values.js**

### Before
Screenshot (76)

### After
Screenshot (77)

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by PrgrmrHarshShukla** [comment]
Hello there, @georgewrmarshall , @brad-decker ,
Kindly look into this PR.

Regards

File: ui/components/app/detected-token/detected-token-values/detected-token-values.js [hunk 1]
```diff
@@ -46,17 +46,18 @@ const DetectedTokenValues = ({
   return (
     <Box display={DISPLAY.INLINE_FLEX} className="detected-token-values">
       <Box marginBottom={1}>
-        <Typography variant={TypographyVariant.H4}>
+        <Text variant={TextVariant.headingSm} as="h4">
```
Comment (georgewrmarshall) [inline ui/components/app/detected-token/detected-token-values/detected-token-values.js:49]: I think this variant matches the font size and weight better

```suggestion
        <Text variant={TextVariant.bodyLgMedium} as="h4">
```

**Review by georgewrmarshall — COMMENTED** [review]
Left one suggestion thanks @PrgrmrHarshShukla

File: ui/components/app/detected-token/detected-token-values/detected-token-values.js (hunk 1 above)
Comment (PrgrmrHarshShukla) [reply ui/components/app/detected-token/detected-token-values/detected-token-values.js:49]: I was also a bit concerned about the font.
👍

**Review by NidhiKJha — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

**Pull request merged**